packages/client/get_audio.py

```diff
@@ -25,7 +25,7 @@ def main():
 
     audio = pyaudio.PyAudio()
     input_device_info = audio.get_device_info_by_index(input_device_index)
-    rate_input = input_device_info.get('defaultSampleRate')
+    rate_input = int(input_device_info.get('defaultSampleRate'))
```

Comment:
Does rate_output not need to be casted to int as well?

Reply:
probably, good catch